packages/renderer/src/lib/markdown/Markdown.spec.ts

```diff
@@ -48,15 +48,17 @@ describe('Custom button', () => {
     await waitRender({ markdown: ':button[Name of the button]' });
     const markdownContent = screen.getByRole('region', { name: 'markdown-content' });
     expect(markdownContent).toBeInTheDocument();
-    expect(markdownContent).toContainHTML('<a class="pf-c-button pf-m-primary" >Name of the button</a>');
+    expect(markdownContent).toContainHTML('Name of the button');
```

Comment:
I would expect that the test check for the html code of the button, not only a name
It could be a label that is set or just the text so we need to ensure a <button> is generated

Reply:
Point taken. I don't love the thought of copying the exact/entire html syntax to the test though. I've added a commit that includes the button and more styling, think this is enough or would you prefer the full html string?